NEWS.md

```diff
@@ -1,5 +1,7 @@
 # vroom (development version)
 
+* Added benchmarks with _wide_ data for both numeric and character data (#87)
```

Comment:
Could you add your GitHub username to the issue? Thanks!

Reply:
Done!